ledger/notary-plugin/src/main/java/net/corda/v5/ledger/notary/plugin/package-info.java

```diff
@@ -0,0 +1,4 @@
+@Export
+package net.corda.v5.ledger.notary.plugin;
+
+import org.osgi.annotation.bundle.Export;
```

Comment:
Is this file necessary, given there's no source files at this level (only under api / core)?

Reply:
Good point!